### ISSUE TITLE
Stop using use-root-reducer?

### ISSUE THREAD

**Issue by dgdavid** [yast/the-installer#13]
[use-root-reducer](https://github.com/ahungrynoob/use-root-reducer) _depends on_ React 16 but the-installer is using React 17. What is more, the package looks small and no so active, reason why I think we can look for the way to stop using it which helps to minimize dependencies (and conflicts).

See https://github.com/yast/the-installer/pull/12

**Comment by dgdavid** [comment]
No longer relevant after #16
